server/src/main/java/com/defold/extender/services/cocoapods/PodSpecParser.java

```diff
@@ -0,0 +1,754 @@
+            String compilerFlag = "";
+            switch (cStandard) {
+                case "ansi":  compilerFlag = "-ansi"; break;
+                case "c89":   compilerFlag = "-std=c89"; break;
+                case "gnu89": compilerFlag = "-std=gnu89"; break;
+                case "c99":   compilerFlag = "-std=c99"; break;
+                case "c11":   compilerFlag = "-std=c11"; break;
+                case "gnu11": compilerFlag = "-std=gnu11"; break;
+                case "gnu99":
+                case "compiler-default": 
+                default:  compilerFlag = "-std=gnu99"; break;
+            }
+            flags.c.add(compilerFlag);
+        }
+        if (hasString(config, "CLANG_CXX_LIBRARY")) {
+            String stdLib = getAsString(config, "CLANG_CXX_LIBRARY", "compiler-default");
+            String stdLibFlag = "";
+            switch (stdLib) {
+                case "libc++": stdLibFlag = "-stdlib=libc++"; break;
+                case "libstdc++":
+                case "compiler-default":
+                default: stdLibFlag = "-stdlib=libstdlibc++"; break;
+            }
+            flags.cpp.add(stdLibFlag);
+            flags.objcpp.add(stdLibFlag);
+        }
+        if (compareString(config, "GCC_ENABLE_CPP_EXCEPTIONS", "YES")) {
+            flags.cpp.add("-fcxx-exceptions");
+        }
+        if (compareString(config, "GCC_ENABLE_CPP_EXCEPTIONS", "NO")) {
+            flags.cpp.add("-fno-cxx-exceptions");
+        }
+        if (compareString(config, "GCC_ENABLE_EXCEPTIONS", "YES")) {
+            flags.add("-fexceptions");
+        }
+        if (compareString(config, "GCC_ENABLE_EXCEPTIONS", "NO")) {
+            flags.add("-fno-exceptions");
+        }
+        if (compareString(config, "GCC_ENABLE_OBJC_EXCEPTIONS", "YES")) {
+            flags.objc.add("-fobjc-exceptions");
+            flags.objcpp.add("-fobjc-exceptions");
+        }
+        if (compareString(config, "GCC_ENABLE_OBJC_EXCEPTIONS", "NO")) {
+            flags.objc.add("-fno-objc-exceptions");
+            flags.objcpp.add("-fno-objc-exceptions");
+        }
+        if (compareString(config, "GCC_ENABLE_CPP_RTTI", "YES")) {
+            flags.cpp.add("-frtti");
+        }
+        if (compareString(config, "GCC_ENABLE_CPP_RTTI", "NO")) {
+            flags.cpp.add("-fno-rtti");
+        }
+        if (compareString(config, "GCC_ENABLE_OBJC_GC", "supported")) {
+            flags.objc.add("-fobjc-gc");
+            flags.objcpp.add("-fobjc-gc");
+        }
+        if (compareString(config, "GCC_ENABLE_OBJC_GC", "required")) {
+            flags.objc.add("-fobjc-gc-only");
+            flags.objcpp.add("-fobjc-gc-only");
+        }
+        if (compareString(config, "GCC_ENABLE_ASM_KEYWORD", "YES")) {
+            flags.add("-fasm");
+        }
+        if (compareString(config, "GCC_ENABLE_ASM_KEYWORD", "NO")) {
+            flags.add("-fno-asm");
+        }
+    }
+
+    static List<String> getStringListValues(JSONObject o, String key) {
```

Comment:
New function which interpret input value as string with ' ' delimeter or as array of string. Also checks if `$(inherited)` is presented and removes it.